.github/workflows/ci-tests.yml

```diff
@@ -2,17 +2,17 @@ name: ci-tests
 on: [push, pull_request]
 permissions: {}
 jobs:
-  ci_tests_ubuntu-18:
-    runs-on: ubuntu-18.04
-    name: Ubuntu-18.04 CI Tests
+  ci_tests_ubuntu-22:
+    runs-on: ubuntu-22.04
+    name: Ubuntu-22.04 CI Tests
```

Comment:
It's uncommon for gnucash to drop support for the previous LTS (20.04 in this case), but I understand why (lack of recent Aqb version).

And honestly I think the linux ecosystem has evolved over the years. The mindset is becoming more one of having a stable base system with current applications. The LTS' increasingly depend on snaps or flatpaks to provide current applications. So to me as application developer it makes less and less sense to keep old code alive for slow-moving distros. I don't think we have to go all in on bleeding edge either, but I'm quite fine with your current choice - aim for the most recent LTS (though even **that** distro version won't ever package what's currently our master branch - it will only go into the **next** LTS).

Reply:
Well, for one thing this isn't about GnuCash supporting an older distro, it's that the older distro image in Github's workflows doesn't provide new enough AQBanking.  A user of that older distro can build a new AQBanking and then build GnuCash 5.

Next problem: We're going to [have to remove](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/) the 18.04 runner in maint in a few weeks; fortunately that's at the end of our dev cycle for the current maint anyway. 20.04 will be similarly killed off in December 2024.

We do need to have the broader discussion about what our dependency requirements policy should be, but I think we should do so on gnucash-devel instead of on a random PR.

Reply:
Ok. If you start that discussion, I'll chime in.

Reply:
> Ok. If you start that discussion, I'll chime in.

Did so.